### DISCUSSION TITLE
weapon changing on steam devel

### DISCUSSION THREAD

**Discussion by nochinator** [redeclipse/base#1402] [Help and Support] [answered]
tell me if I missed something obvious but I can not figure out how to change loadout on devel, is it not implemented for the new version yet?

**Comment by dc-redeclipse** [comment, marked as answer]
As I said the UI is under deeper development at the moment but this is nearly finish.
![20230128_233546](https://user-images.githubusercontent.com/60038236/215294231-5d8841a1-64bd-4ce0-8299-e34a9a0fd603.png)
Just click in menu on Player Setup and you will find the Loadout.

**Reply by nochinator** [reply]
yeah sorry for the spam here I just couldn't figure it out, thank you very much.

**Reply by dc-redeclipse** [reply]
No problem! You are welcome and always free to ask. For those miner questions you can just ask those on Red Eclipse on our discord if you like to. This might be an much more efficient way for quick help sometimes.
